src/backend/gl/src/command.rs

```diff
@@ -454,8 +466,10 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
         self.push_cmd(Command::BindIndexBuffer(ibv.buffer.raw));
     }
 
-    fn bind_vertex_buffers(&mut self, _vbs: hal::pso::VertexBufferSet<Backend>) {
-        unimplemented!()
+    fn bind_vertex_buffers(&mut self, vbs: hal::pso::VertexBufferSet<Backend>) {
+        for vertex_buffer in vbs.0 {
+            self.push_cmd(Command::BindVertexBuffer(vertex_buffer.0.raw));
```

Comment:
This would work if `BindVertexBuffer` would actually be translated to a `glBindVertexBuffer` call, but atm it only overrides the buffer bound at `ARRAY_BUFFER`.

Reply:
True, maybe it's better to simply no-op this call for now until we add `glBindVertexBuffer`. At the moment it will be handled in `BindAttribute` anyway